model-optimizer/extensions/ops/scatter.py

```diff
@@ -94,6 +96,41 @@ class ScatterElementsUpdate(Scatter):
     op = op_type = 'ScatterElementsUpdate'
     version = 'opset3'
 
+    @staticmethod
+    def infer(node: Node):
+        Scatter.infer(node)
+
+        node_name = node.soft_get('name', node.id)
+        connected_in_ports = [port for port in node.in_ports().values() if not port.disconnected()]
+        assert len(connected_in_ports) == 4, \
+            "Incorrect number of inputs for {} node".format(node_name)
+
+        input_value = node.in_port(0).data.get_value()
+        input_shape = node.in_port(0).data.get_shape()
+        indices_value = node.in_port(1).data.get_value()
+        indices_shape = node.in_port(1).data.get_shape()
+        updates_value = node.in_port(2).data.get_value()
+        updates_shape = node.in_port(2).data.get_shape()
+        axis = node.in_port(3).data.get_value()
+
+        data_rank = len(input_shape)
+        assert data_rank == len(indices_shape), 'data and indices inputs for node {} must be of the ' \
+                                                'same rank. Instead got {} and {}'. \
+            format(node.name, data_rank, len(indices_shape))
+
+        assert np.array_equal(indices_shape, updates_shape), 'updates and indices shapes for node {} must be equal. ' \
+                                                             'Instead got {} and {}'. \
+            format(node.name, indices_shape, updates_shape)
+
+        # compute output value if all inputs are constant
+        if input_value is not None and indices_value is not None and updates_value is not None:
+            out_value = input_value.copy()
+            for idx in np.ndindex(*indices_shape):
+                data_idx = list(idx)
+                data_idx[axis] = indices_value[idx]
```

Comment:
What if the `axis` value is a tensor of shape `[1]` instead of scalar? I think that this line will not work in this case. So, let's have an assert that `axis.size == 1` and here use `axis.item()` to get the only element of the `axis` tensor.